src/liballoc/str.rs

```diff
@@ -2140,48 +2140,6 @@ impl str {
         unsafe { String::from_utf8_unchecked(buf) }
     }
 
-    /// Returns true if this `str` is entirely whitespace, and false otherwise.
-    ///
-    /// 'Whitespace' is defined according to the terms of the Unicode Derived Core
-    /// Property `White_Space`.
-    ///
-    /// # Examples
-    ///
-    /// Basic usage:
-    ///
-    /// ```
-    /// assert!("    \t ".is_whitespace());
-    ///
-    /// // a non-breaking space
-    /// assert!("\u{A0}".is_whitespace());
-    ///
-    /// assert!(!"   越".is_whitespace());
-    /// ```
-    #[stable(feature = "unicode_methods_on_intrinsics", since = "1.27.0")]
-    #[inline]
-    pub fn is_whitespace(&self) -> bool {
```

Comment:
Should we be destabilizing and deprecating these for a bit? Not sure if they've been around and unstable previously.

Reply:
I'd be ok with that as well yeah, or perhaps just reverting these to unstable? I'm fine either way, they were added insta-stable in https://github.com/rust-lang/rust/pull/49381 and haven't been around for all that long so I don't think we need too too much process

Reply:
Oh cool if they're new then it seems fine to just remove.